modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -39,6 +39,43 @@ This column is returned by default.
 
 The column returns the name of the composite database that the alias belongs to, or `null` if the alias does not belong to a composite database.
 
+
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+IS [NOT] :: <TYPE>
+----
+a|
+
+Extended xref:syntax/expressions.adoc#type-predicate-expressions[type predicate expressions].
+Closed dynamic union types (`type1 \| type2 \| ...`) are now supported, allowing for queries such as:
```

Comment:
```suggestion
Closed dynamic union types (`type1 \| type2 \| ...`) are now supported, allowing for expressions such as:
```

Reply:
This is part of Gem's changes, so not really relevant here